test/functionalTest/cases/3176_entity_without_attributes_not_listed/entity_without_attributes_not_listed.test

```diff
@@ -0,0 +1,126 @@
+# Copyright 2018 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
This is the same test (or a trivial variation of it) that @jmcanterafonseca provided in the parent issue?

Reply:
The exact same.

NTC